streams/src/test/java/org/apache/kafka/streams/integration/MetricsIntegrationTest.java

```diff
@@ -207,17 +241,22 @@ private void checkMetricDeregistration() throws InterruptedException {
     @Test
     public void testStreamMetric() throws Exception {
         final StringBuilder errorMessage = new StringBuilder();
-        stream = builder.stream(STREAM_INPUT, Consumed.with(Serdes.Integer(), Serdes.String()));
-        stream.to(STREAM_OUTPUT_1, Produced.with(Serdes.Integer(), Serdes.String()));
+        builder.stream(STREAM_INPUT, Consumed.with(Serdes.Integer(), Serdes.String()))
+            .to(STREAM_OUTPUT_1, Produced.with(Serdes.Integer(), Serdes.String()));
         builder.table(STREAM_OUTPUT_1, Materialized.as(Stores.inMemoryKeyValueStore(MY_STORE_IN_MEMORY)).withCachingEnabled())
                 .toStream()
                 .to(STREAM_OUTPUT_2);
         builder.table(STREAM_OUTPUT_2, Materialized.as(Stores.persistentKeyValueStore(MY_STORE_PERSISTENT_KEY_VALUE)).withCachingEnabled())
                 .toStream()
                 .to(STREAM_OUTPUT_3);
-        builder.table(STREAM_OUTPUT_3, Materialized.as(Stores.lruMap(MY_STORE_LRU_MAP, 10000)).withCachingEnabled())
-                .toStream()
-                .to(STREAM_OUTPUT_4);
+        final KGroupedStream<Integer, Integer> groupedStream = builder.stream(STREAM_OUTPUT_3, Consumed.with(Serdes.Integer(), Serdes.Integer()))
```

Comment:
This change is intentional: to check suppression-emit and skipped-records sensors which are created lazily we need to make topology a bit more complex.